staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1249,11 +1255,15 @@ func (c *Cacher) LastSyncResourceVersion() (uint64, error) {
 // spits a ResourceVersion after which the bookmark event will be delivered.
 //
 // The returned function must be called under the watchCache lock.
-func (c *Cacher) getBookmarkAfterResourceVersionLockedFunc(ctx context.Context, parsedResourceVersion uint64, opts storage.ListOptions) (func() uint64, error) {
+func (c *Cacher) getBookmarkAfterResourceVersionLockedFunc(parsedResourceVersion, watchCacheResourceVersion uint64, opts storage.ListOptions) (func() uint64, error) {
```

Comment:
I don't like the name of the `watchCacheResourceVersion` param as it can be confused with`c.watchCache.resourceVersion`. Any ideas ? Maybe we should rename it to `storageResourceVersion` or `externalResourceVersion` ?

Reply:
requiredResourceVersion maybe?

[it's effectively the RV that we need to be synced to]?